EET-216 Retire ZW sample dev token (#222)
zelloptt/zello-channel-api: API.md

This API supports subset of Zello features and currently focused on sending and receiving channel voice messages. See [Supported features](#supported-features) for the complete list.

> To access the API you need to generate a valid access token (optional for Zello Work), based on [JWT](https://jwt.io/) standard. See [Authentication](#authentication).

Reviewer: ```suggestion
To access the API you will need valid account credentials and/or a valid access token, based on the [JWT](https://jwt.io/) standard. See [Authentication](#authentication).
```

Reviewer: please also remove the reference to tokens for Zello Work on line 40 below

Author: Changed
